src/api/file_storage.rs

```diff
@@ -42,17 +49,114 @@ pub fn file_storage_scope() -> Scope {
       web::resource("/{workspace_id}/blobs")
         .route(web::get().to(get_all_workspace_blob_metadata_handler)),
     )
+    .service(web::resource("/{workspace_id}/create_upload").route(web::post().to(create_upload)))
```

Comment:
there's `workspace_id` in the path, it should be enforced and used as prefix for object key in the server, else this allow client to store in object using any key, potentially affecting other user

Reply:
It creates a upload id. The object_key will be used in the upload part api endpoint